packages/astro/src/actions/runtime/virtual/shared.ts

```diff
@@ -204,14 +204,26 @@ export function serializeActionResult(res: SafeResult<any, any>): SerializedActi
 		if (import.meta.env?.DEV) {
 			actionResultErrorStack.set(res.error.stack);
 		}
+
+		let body: Record<string, any>;
+		if(res.error instanceof ActionInputError) {
+			body = {
+				type: res.error.type,
+				issues: res.error.issues,
+				fields: res.error.fields
```

Comment:
Interesting that we need to parse out `issues` and `fields` manually here. What was it passing before?

Reply:
The entire error object which includes the `stack` and the `message` both of which can be quite long, and including either results in this test being too large.